printer/Brother-PT-9700PC.xml

```diff
@@ -0,0 +1,79 @@
+<!--
+Copyright (c) 2006  Arne John Glenstrup <panic@itu.dk>
+
+This file is part of ptouch-driver.
+
+ptouch-driver is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 2 of the License, or (at
+your option) any later version.
+
+ptouch-driver is distributed in the hope that it will be useful, but
+WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with ptouch-driver; if not, write to the Free Software
+Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301
+USA
+-->
+<printer id="printer/Brother-PT-9700PC">
+  <make>Brother</make>
+  <model>PT-9700PC</model>
+  <pcmodel>BR950P</pcmodel>
+  <mechanism>
+    <thermal/>
+    <!--not "color"-->
+    <resolution>
+      <dpi>
+        <x>360</x>
+        <y>720</y>
+      </dpi>
+    </resolution>
+  </mechanism>
+  <url>https://support.brother.com/g/b/producttop.aspx?c=us_ot&amp;lang=en&amp;prod=9700eus</url>
+  <lang>
+    <proprietary />
+  </lang>
+  <autodetect>
+    <general>
+      <ieee1284>MFG:Brother;CMD:PT-CBP;MDL:PT-9700PC;CLS:PRINTER;</ieee1284>
```

Comment:
Does "./getusbprinterid /dev/usb/lp0" really identify the printer model as "MDL:PT-P9700PC"?

Reply:
yep:
```
$ sudo ./getusbprinterid /dev/usb/lp0
DeviceID MFG:Brother;CMD:PT-CBP;MDL:PT-9700PC;CLS:PRINTER;
Vendor '1273', Product '8252'
$
```